cfme/automate/dialog_collection_pick.py

```diff
@@ -1,9 +0,0 @@
-from widgetastic.utils import VersionPick
-
-from cfme.automate.service_dialogs import DialogCollection as dc58  # noqa
-from cfme.automate.dialogs.service_dialogs import DialogCollection as dc59  # noqa
-from cfme.utils import version
-
-collection_pick = VersionPick({
-    version.LOWEST: dc58,
-    '5.9': dc59})
```

Comment:
There are dialog code files for both 5.8 and 5.9 that gets picked by this code . If we are dropping automation for 5.8 then we will have to remove those files too .

Reply:
Done

Reply:
@quarckster : Also please remove automate/dialog_tab.py , dialog_element.py and dialog_box.py

Reply:
Done